lib/internal/Magento/Framework/App/MaintenanceMode.php

```diff
@@ -44,14 +45,22 @@ class MaintenanceMode
      */
     private $eventManager;
 
+    /**
+     * @var IpChecker
+     */
+    private $ipChecker;
+
     /**
      * @param \Magento\Framework\Filesystem $filesystem
      * @param Manager|null $eventManager
+     * @param IpChecker|null $ipChecker
+     * @throws \Magento\Framework\Exception\FileSystemException
```

Comment:
For the new submissions, we facilitate using imports instead of FQN. Could you replace the FQN of the class with an import, please?

Reply:
done